contents/blog/best-hotjar-alternatives.md

```diff
@@ -0,0 +1,431 @@
+---
+title: The best Hotjar alternatives & competitors, compared
+date: 2024-03-04
+author: ["andy-vandervell"]
+rootpage: /blog
+featuredImage: ../images/blog/hotjar-alternatives/hotjar-alternatives.png
+featuredImageType: full
+category: General
+tags:
+ - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+## 1. PostHog
+
+- **Founded:** 2020
+- **Similar to:** LogRocket, Mixpanel
+- **Typical users:** Engineers and product teams
+- **Typical customers:** Mid-size B2B/B2C companies and startups
+
+![posthog](../images/screenshots/session-replay.png)
+
+### What is PostHog?
+
+[PostHog](/) (that's us 👋) is an all-in-one platform for session replay, product analytics, feature flags, A/B testing, and user surveys – everything you need to understand your users. This means it's not just an alternative to Hotjar, but also tools like [Mixpanel](/blog/best-mixpanel-alternatives) and [LaunchDarkly](/blog/best-launchdarkly-alternatives).
+
+According to [BuiltWith](https://trends.builtwith.com/analytics/PostHog), as of February 2024, PostHog is used on 5,169 (0.52%) of the top 1 million websites. Hotjar is used by [72,048 of the top 1 million websites](https://trends.builtwith.com/analytics/Microsoft-Clarity). Typical PostHog users are engineers and product managers at startups and mid-size companies, such as [Webshare](/customers/webshare), [AssemblyAI](/customers/assemblyai), and [Purplewave](/customers/purplewave).
```

Comment:
> According to [BuiltWith](https://trends.builtwith.com/analytics/PostHog), as of February 2024, PostHog is used on 5,169 (0.52%) of the top 1 million websites.

I get we're trying to built credibilty for ourselves, but Im not sure this comes off as super credible. I think theres something better we could do here

Reply:
Yeah, the main purpose of this is kind of more SEO orientated, it signals to Google we're citing some actual sources rather than just saying "this is what we think". It's the reason we refer to "according to G2". BuiltWith isn't the best source, but it's as good as we have. 